packages/core/src/amazonqFeatureDev/util/files.ts

```diff
@@ -49,6 +60,17 @@ export async function prepareRepoData(
             zip.addLocalFile(file.fileUri.fsPath, zipFolderPath)
         }
 
+        ignoredExtensionMap.forEach(async (value, key) => {
```

Comment:
why not use a regular for-loop? if this async callback fails, will it correctly bubble up an exception? that is one reason why forEach() can lead to hidden bugs.

Reply:
ok, will make a change